ui/src/app/components/lists.tsx

```diff
@@ -3,9 +3,13 @@ import * as React from 'react';
 export const CustomBulletList = ({children}) => {
   return <ul style={{position: 'relative', listStyle: 'none', marginLeft: 0}}>{children}</ul>;
 };
-
 export const CustomBulletListItem = ({bullet, children}) => {
   return <React.Fragment>
-    <div style={{position: 'absolute', left: '-.2em'}}> {bullet} </div> <li style={{marginLeft: '1.2em'}}> {children} </li>
+    <div style={{position: 'absolute', left: '-.2em'}}> {bullet} </div>
```

Comment:
Was this just split into two lines opportunistically for readability?

Reply:
Yes